Mathlib/Geometry/Euclidean/Incenter.lean

```diff
@@ -658,6 +658,22 @@ lemma isTangentAt_insphere_iff_eq_touchpoint {i : Fin (n + 1)} {p : P} :
       p = s.touchpoint ∅ i :=
   s.excenterExists_empty.isTangentAt_exsphere_iff_eq_touchpoint
 
+variable {s} in
+lemma ExcenterExists.affineSpan_faceOpposite_eq_orthRadius [hf : Fact (Module.finrank ℝ V = n)]
+    {signs : Finset (Fin (n + 1))} (h : s.ExcenterExists signs) (i : Fin (n + 1)) :
+    affineSpan ℝ (Set.range (s.faceOpposite i).points) =
+      (s.exsphere signs).orthRadius (s.touchpoint signs i) := by
+  refine (h.isTangentAt_touchpoint i).eq_orthRadius_of_finrank_add_one_eq (h.exradius_pos.ne') ?_
+  rw [direction_affineSpan, (s.faceOpposite i).independent.finrank_vectorSpan_add_one,
+    Fintype.card_fin, hf.out]
+  have := NeZero.ne n
+  cutsat
+
+lemma affineSpan_faceOpposite_eq_orthRadius_insphere [Fact (Module.finrank ℝ V = n)]
+    (i : Fin (n + 1)) :
+    affineSpan ℝ (Set.range (s.faceOpposite i).points) = s.insphere.orthRadius (s.touchpoint ∅ i) :=
```

Comment:
If we do opt for the spelling above, then this should be able to carry the `simp` attribute:
```suggestion
@[simp] lemma affineSpan_points_compl_eq_orthRadius_insphere [Fact (Module.finrank ℝ V = n)]
    (i : Fin (n + 1)) :
    affineSpan ℝ (s.points '' {i}ᶜ) = s.insphere.orthRadius (s.touchpoint ∅ i) :=
```

Reply:
I don't think this (translating from a purely affine reference to the span of a face, to something involving the insphere which you might not care about at all in that context) would be a good `simp` lemma. The reverse direction would be more plausible as a `simp` lemma, but I still think it would be too contextual whether "span of face" or "tangent at touchpoint" is the form that is wanted.